src/main/java/globalquake/ui/database/SeedlinkServersPanel.java

```diff
@@ -20,6 +20,8 @@ public class SeedlinkServersPanel extends JPanel {
     private final EditSeedlinkNetworkAction editSeedlinkNetworkAction;
     private final RemoveSeedlinkNetworkAction removeSeedlinkNetworkAction;
     private final UpdateSeedlinkNetworkAction updateSeedlinkNetworkAction;
+    private JButton btnSelectStations;
+    private JButton btnLaunch;
```

Comment:
Same here... better to use the getters directly as these fields are used only once